sdk/src/wallet/storage/manager.rs

```diff
@@ -182,3 +182,64 @@ impl StorageManager {
             .await
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::wallet::storage::adapter::memory::{Memory, STORAGE_ID};
+
+    #[tokio::test]
+    async fn id() {
+        let storage_manager = StorageManager::new(None, Box::<Memory>::default()).await.unwrap();
+        assert_eq!(storage_manager.id(), STORAGE_ID);
+        assert!(!storage_manager.is_encrypted());
+    }
+
+    #[tokio::test]
+    async fn get() {
+        #[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
+        struct Record {
+            a: String,
+            b: u32,
+            c: i64,
+        }
+
+        let rec = Record {
+            a: "test".to_string(),
+            b: 42,
+            c: -420,
+        };
+        let mut storage = Box::<Memory>::default();
+        storage.set("key", serde_json::to_string(&rec).unwrap()).await.unwrap();
```

Comment:
You could have these tests return an option for convenience sake.

Reply:
Not sure what you mean by "tests return an option for convenience" here tbh ... but since it's just for convenience I hope it's not something you really insist on, so I am marking this as resolved.